### PR TITLE
Fixed issue #13660: Core question types & themes are not translated in question selector

### PR DESCRIPTION
**Pull request by c-schmitz**

### CODE REVIEW

**Comment by Shnoulle** [comment]
I'm really happy to don't use this constant. This potentially broke plugins

It's only for core question theme here ? Still no way for User question theme ?

**Comment by olleharstedt** [comment]
> I'm really happy to don't use this constant. This potentially broke plugins

We were just discussing this.

**Comment by olleharstedt** [comment]
Translation of user question theme is another topic, yes.

**Comment by Shnoulle** [comment]
> Translation of user question theme is another topic, yes.

OK :)

File: application/models/QuestionType.php [hunk 1]
```diff
@@ -46,6 +46,22 @@ class QuestionType extends StaticModel
     const QT_COLON_ARRAY_NUMBERS = ':';
     const QT_SEMICOLON_ARRAY_TEXT = ';';
 
+    /*
+    * @deprecated The following constants are deprecated and will be removed in LimeSurvey 6 - please use the ones above.
```
Comment (Shnoulle) [inline application/models/QuestionType.php:50]: :+1:

**Comment by olleharstedt** [comment]
This test fails now:

>  ✘ Get question theme on saved question

**Comment by c-schmitz** [comment]
Failed test is fixed.

**Pull request merged**